ui/components/app/name/name.tsx

```diff
@@ -6,27 +6,18 @@ import React, {
   useState,
 } from 'react';
 import { NameType } from '@metamask/name-controller';
-import classnames from 'classnames';
-import { toChecksumAddress } from 'ethereumjs-util';
-import { Box, Icon, IconName, IconSize, Text } from '../../component-library';
-import { shortenAddress, shortenString } from '../../../helpers/utils/util';
+import { Box } from '../../component-library';
 import { MetaMetricsContext } from '../../../contexts/metametrics';
 import {
   MetaMetricsEventCategory,
   MetaMetricsEventName,
 } from '../../../../shared/constants/metametrics';
-import { Display, TextVariant } from '../../../helpers/constants/design-system';
+import { Display } from '../../../helpers/constants/design-system';
 import { useDisplayName } from '../../../hooks/useDisplayName';
-import Identicon from '../../ui/identicon';
+import NameDisplay from './name-details/name-display';
 import NameDetails from './name-details/name-details';
 
 export type NameProps = {
-  /** Whether to prevent the modal from opening when the component is clicked. */
-  disableEdit?: boolean;
-
-  /** Whether this is being rendered inside the NameDetails modal. */
-  internal?: boolean;
-
```

Comment:
The `disableEdit` and `internal` properties were only being used because the component circularly referenced and partially re-rendered itself. Now that the part that was re-rendered is its own component (`name-display.tsx`), we have no use for these properties. So they've been removed.